src/wrappers/pubsub.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable prefer-rest-params */
+/**
+ * @fileoverview Handlers for Pubsub instrumentation
+ */
+
+const shimmer = require('shimmer');
+const {
+    tracer,
+    tryRequire,
+} = require('epsagon');
+const { initializeEvent, finalizeEvent } = require('../runners/runner');
+const traceContext = require('../trace_context.js');
+
+const GOOGLE_CLOUD_TYPES = {
```

Comment:
again, looks like a duplication

Reply:
I don't want to create another types file and export him in epsagon-node, I'm only used 'pubsub' stirng. So I removed GOOGLE_CLOUD_TYPES.